src/content/documentation/publish/self-distribution.md

```diff
@@ -61,6 +61,52 @@ When you choose to distribute an add-on yourself, they can be installed using th
   content: content
 %}
 
+{% capture content %}
+
+### Web download
+
+When distributing an add–on from your website, you have two options for making the signed add–on file available for download:
+
+- From a link on a webpage, like this:
+  ```html
+  <div id="example-option-1" class="install-ok">
+    <a href="https://addons.mozilla.org/firefox/downloads/file/3923300/facebook_container-2.3.2-fx.xpi">
```

Comment:
@rebloor given that this page is related to self-distribution, I think it may be reasonable to use a fake xpi url that does point to a website that isn't addons.mozilla.org, e.g. "https://example.com/path/to/extension.xpi", wdyt?

Reply:
Done